src/content/release-notes.mdx

```diff
@@ -5,6 +5,29 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.16.0
+
+20 December 2023
+
+### Improvements
```

Comment:
Can we standarize in the order of the sections? What we have used more times is:
- Kubernetes versions supported
- Breaking changes
- Deprecations
- Features
- Improvements
- Bugfixes
Also, we are supporting k8s 1.28 now, right?

Reply:
Yes, we start supporting `1.28` as part of this release, and also, this is the first release where we start supporting 4 k8s versions,, right?